pkg/volume/plugins.go

```diff
@@ -49,6 +49,8 @@ type VolumeOptions struct {
 	// PV.Name of the appropriate PersistentVolume. Used to generate cloud
 	// volume name.
 	PVName string
+	// PVC.Name of the PersistentVolumeClaim, if we are auto-provisioning.
```

Comment:
All of VolumeOptions is only used for auto-provisioning.  I guess "if we are auto-provisioning" is technically superfluous; I thought it was clearer but I guess it begs the question of when VolumeOptions is used other than auto-provisioning.